### PR TITLE
A user can see score 2

### PR DESCRIPTION
**Pull request by mohammadalnajar**

### CODE REVIEW

File: src/views/createQuestionElement.js [hunk 1]
```diff
@@ -13,18 +17,19 @@ export const createQuestionElement = (question) => {
...
-      console.log(e);
       clickCounter++;
-
+      // question.selected = e.target.dataset.answerItemKey; discuss this line with the group
       if (
         e.target.dataset.answerItemKey == question.correct &&
         clickCounter == 1
       ) {
         e.target.classList.add('correct');
+        numberOfCorrectAnswers.push(question);
```
Comment (wilgert) [inline src/views/createQuestionElement.js:27]: Rename to `correctAnswers`

File: src/views/createQuestionElement.js [hunk 2]
```diff
@@ -13,18 +17,19 @@ export const createQuestionElement = (question) => {
...
       } else if (
         e.target.dataset.answerItemKey != question.correct &&
         clickCounter == 1
       ) {
+        numberOfWrongAnswers.push(question);
```
Comment (wilgert) [inline src/views/createQuestionElement.js:32]: Rename to `wrongAnswers`

File: src/handlers/handleNextQuestion.js [hunk 3]
```diff
@@ -1,24 +1,76 @@
 'use strict';
 
-import showCurrentQuestion from './showCurrentQuestion.js';
+import { showCurrentQuestion } from './showCurrentQuestion.js';
 import { quizData } from '../data.js';
+import createDOMElement from '../utils/createDOMElement.js';
+import { questionContainer } from '../handlers/showCurrentQuestion.js';
+import {
```
Comment (wilgert) [inline src/handlers/handleNextQuestion.js:7]: Get `score` and `correctAnswers` from `quizData`

File: src/data.js [hunk 4]
```diff
@@ -35,6 +35,110 @@ export const quizData = {
...
+    {
+      text: 'Where is the correct place to insert a JavaScript?',
+      answers: {
+        a: 'Both the <head> section and the <body> section are correct',
+        b: 'The <body> section',
+        c: 'The <head> section',
+      },
+      correct: 'b',
+      selected: null,
+      links: [
```
Comment (wilgert) [inline src/data.js:68]: Add the relevant links

File: src/handlers/handleNextQuestion.js [hunk 5]
```diff
@@ -1,24 +1,76 @@
...
+        const correctQuestion = createDOMElement('h2');
+        const correctAnswer = createDOMElement('h4');
+        correctQuestion.innerText = `${i + 1}- ${item.text}`;
+
+        newQuestionContainer.appendChild(correctQuestion);
+
+        //loop through the answers object to get only the correct answer
+        for (const answerKey in item.answers) {
+          if (item.correct == answerKey) {
+            console.log(item.answers[answerKey]);
```
Comment (wilgert) [inline src/handlers/handleNextQuestion.js:55]: remove `console.log`

**Review by wilgert — CHANGES_REQUESTED** [review]

File: src/views/createQuestionElement.js (hunk 1 above)
Comment (arsalanamanat) [reply src/views/createQuestionElement.js:27]: Done....

File: src/views/createQuestionElement.js (hunk 2 above)
Comment (arsalanamanat) [reply src/views/createQuestionElement.js:32]: Done

File: src/handlers/handleNextQuestion.js (hunk 3 above)
Comment (arsalanamanat) [reply src/handlers/handleNextQuestion.js:7]: Modifications Done. Its not possible for get Score from QuizData . As we are creating a new Element of <H3>

File: src/data.js (hunk 4 above)
Comment (arsalanamanat) [reply src/data.js:68]: Done...

File: src/views/createQuestionElement.js [hunk 6]
```diff
@@ -2,15 +2,17 @@
...
 
-export const score = createDOMElement('h3', { className: 'user-score' });
 export const createQuestionElement = (question) => {
   const container = createDOMElement('div');
   const title = createDOMElement('h1');
   title.innerText = question.text;
   container.appendChild(title);
   const answerContainer = createDOMElement('ol');
   let clickCounter = 0;
+  const score = createDOMElement('h3', { id: 'user-score' });
```
Comment (wilgert) [inline src/views/createQuestionElement.js:14]: I didn't realize that `score` was an html element. Maybe you can change the name to `scoreElement`

**Review by wilgert — APPROVED** [review]

**Pull request merged**